### PR TITLE
BOM-1968 : Post Python 3.8 changes

### PR DESCRIPTION
**Pull request by mzulqarnain1**
Relevant JIRA : https://openedx.atlassian.net/browse/BOM-1968

### CODE REVIEW

File: requirements/all.txt [hunk 1]
```diff
@@ -124,7 +121,6 @@ python-memcached==1.59    # via -r requirements/dev.txt, -r requirements/product
 python-slugify==4.0.1     # via -r requirements/dev.txt, transifex-client
 python3-openid==3.2.0     # via -r requirements/dev.txt, -r requirements/production.txt, social-auth-core
 pytz==2020.1              # via -r requirements/dev.txt, -r requirements/production.txt, django, django-ses
-pywatchman==1.4.1 ; "linux" in sys_platform  # via -r requirements/dev.txt
```
Comment (jmbowman) [inline requirements/all.txt:127]: I think we still want this for devstack CPU usage minimization.

Comment (mzulqarnain1) [reply requirements/all.txt]: Added back.

**Comment by mzulqarnain1** [comment]
> Looks good to me, created [EDUCATOR-5238](https://openedx.atlassian.net/browse/EDUCATOR-5238) for review by the owning team.
> 
> Were the classifier updates in setup.py forgotten or deliberately deferred for a later PR?

@jmbowman There is not `setup.py` file in this repo.

**Review by tuchfarber — APPROVED** [review]
Looks good. We're testing Django 2.2, 3.0, and 3.1 in tox, but only 2.2, and 3.0 in Travis? Can we sync these up or is there a reason we're not doing 3.1 in Travis?

**Comment by mzulqarnain1** [comment]
> Looks good. We're testing Django 2.2, 3.0, and 3.1 in tox, but only 2.2, and 3.0 in Travis? Can we sync these up or is there a reason we're not doing 3.1 in Travis?

@tuchfarber I've added Django 3.1 in travis.

**Pull request merged**